### DISCUSSION TITLE
Not able to debug CSS: source maps missing and only main.css generated. Nx + Angular.

### DISCUSSION THREAD

**Discussion by korjikk** [storybookjs/storybook#26668] [Help]
### Summary

Hi! Did anyone run across an issue with debugging styles when using Nx + Angular + Storybook? I cannot see source maps (from what stylesheet the style is coming), as there's only main.css generated

**Comment by Fernker** [comment]
Hi there, I had this issue last year and posted about it on the Discord. 

Here's the thread:
https://discord.com/channels/486522875931656193/1117856985770102904

We ended up using the npm `patch-package` package to overwrite the issue in the storybook files.  As we couldn't get more configs in `angular.json` working and it was a wrong solution for the problem. 

Our patch looks like this:
![Screenshot 2024-03-28 at 4 20 21 PM](https://github.com/storybookjs/storybook/assets/2066227/c097ff18-7d0e-4e5c-b123-cfc03cd4f373)

I'm at a different company now and haven't gotten back to seeing if Storybook@v8 improves this.
